schema.sql

```diff
@@ -6,7 +6,8 @@ CREATE TABLE animals (
     date_of_birth DATE,
     escape_attempts INT,
     neutered BOOLEAN,
-    weight_kg FLOAT
+    weight_kg FLOAT,
+    species VARCHAR(100)
```

Comment:
- [ ] It's nice if you do table modifications on a separate statement, don't edit already ran queries:
`ALTER TABLE animals ADD species varchar(255);`

Reply:
Hi @descholar-ceo ,
Thanks for the feedback, i'll do that.